integration_tests/runtests.py

```diff
@@ -22,4 +23,16 @@
     #cProfile.run('main(sys.argv[1:])', 'test_profile')
     # arguments are the same as for pytest https://docs.pytest.org/en/latest/usage.html
     # or run pytest -h
-    sys.exit(main(sys.argv[1:]))
+    iteration = 0
```

Comment:
In the documentation at https://github.com/NVIDIA/spark-rapids/blob/branch-0.4/integration_tests/README.md it says

```
The only reason we have a separate script is that spark-submit uses python if the file name ends with .py.
``` 

After this change it is no longer true.  If you decide not to use `pytest-repeat` could you please update the documentation to explain how this works and remove the comment about why we created this script.